images/Dockerfile.install

```diff
@@ -1,10 +1,11 @@
 # Copyright (c) Microsoft Corporation
 # SPDX-License-Identifier: MIT
 
-ARG WINDOWS_VERSION=1809
+# Base image for Windows nanoserver 1809.
+ARG WINDOWS_IMAGE_HASH=sha256:fb16d42712f3599c849723d365a7ddc5fb95e8bf1674d97e9f9cbcb726e7525d
 
 # The files in this image are copied to $env:CONTAINER_SANDBOX_MOUNT_POINT on the host.
-FROM mcr.microsoft.com/windows/nanoserver:${WINDOWS_VERSION}
+FROM mcr.microsoft.com/windows/nanoserver@${WINDOWS_IMAGE_HASH}
```

Comment:
the code scanning scorecard still thinks this isn't pinned by hash for some reason.  I wonder if it needs the hash to be literal rather than saved in a variable?